pykube/objects.py

```diff
@@ -131,18 +131,24 @@ def watch(self):
             "metadata.name": self.name
         }).watch()
 
-    def update(self):
+    def patch(self, patch):
```

Comment:
`kubectl patch` supports multiple patch formats. It might be more self-explanatory (and more practical for future extension to further patch formats) to rename the `patch` parameter:
```suggestion
    def patch(self, merge_patch):
```